### DISCUSSION TITLE
[quadlet] No way to specify SOCKS5 proxy

### DISCUSSION THREAD

**Discussion by jcpunk** [containers/podman#21387] [General]
### Feature request description

Some of my hosts require use of a SOCKS5 proxy to reach outside their protected network (and to my private registry).  Since all the `Environment` options in the quadlet config are for the container, there isn't a way to set the PROXY environment variables for the systemd service.

### Suggest potential solution

Some obvious way to setup the PROXY environment options for the service itself so podman knows to use my SOCKS5 proxy.

### Have you considered any alternatives?

Hacking up some systemd dropins

### Additional context

There doesn't appear to be a setting in `registries.conf` for this either.

**Comment by rhatdan** [comment]
You should be able to set 
[Service]

Content which systemd will use.
